.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/ruby
 
 # Generated js bundles
 /app/assets/javascripts/generated/*
+
+# Rubymine/IntelliJ
+.idea
```

Comment:
I agree with @geoffevason about having some of this in the project .gitignore. For this particular reason:

  98% of the developers are using OS X, and we keep having this discussion about adding theses files to the global .gitignore. During different PR's and projects. 

Lets just add the specifc OS X files and be done. The amount of time and energy to keep revisiting this is a waste of everyone's valuable time.

Just my two cents @samnang @justin808